src/system.jl

```diff
@@ -20,6 +20,8 @@ struct Zone
     good_utility::Float64
 end
 
+const Zones = Dictionary{Int, Zone}
```

Comment:
perhaps let's have the key types be a const too so we can set it once and use it both here and in the type, so we only have to change it in one place in future -- what do you think?

e.g.
```julia
const ZoneNum = Int64
struct Zone
    "Zone number"
    number::ZoneNum
    ...
end
const Zones = Dictionary{ZoneNum, Zone}
```

Reply:
Sure, that's already the case for Branches and Buses, makes sense to do it for Zones and Generators too.